implement-shell-tools/ls/ls.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import os 
+
+parser = argparse.ArgumentParser(
+                    prog='ls command',
+                    description='list directory contents')
+
+parser.add_argument("-1", "--one", help="Force output to be one entry per line.", default = None, action="store_true")
+parser.add_argument("-a", help="Include directory entries whose names begin with a dot (.).", default = None, action="store_true")
```

Comment:
When I run the `ls` utility with `-a`, the current directory (`.`) and parent directory (`..`) are printed:

```shell
$ ls -a sample-files
.               ..              .hidden.txt     1.txt           2.txt           3.txt           dir
```

How could you change your solution to make sure they are printed?